crates/sui-indexer/migrations/2023-03-05-165748_transaction_index/up.sql

```diff
@@ -24,6 +25,7 @@ CREATE TABLE recipients (
 CREATE INDEX recipients_transaction_digest ON recipients (transaction_digest);
 CREATE INDEX recipients_sender ON recipients (sender);
 CREATE INDEX recipients_recipient ON recipients (recipient);
+CREATE INDEX recipients_checkpoint ON recipients (checkpoint_sequence_number);
```

Comment:
this is not for this view -> table change, but for address_processor to read recipients instead in https://github.com/MystenLabs/sui/pull/13513